### PR TITLE
semgrep 0.76.0

### PR DESCRIPTION
**Pull request by pthariensflame**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
```
Error: Program pipenv not found in the tree or in PATH
```
@brendongo is that a new dependency?

File: Formula/semgrep.rb [hunk 1]
```diff
@@ -117,11 +116,6 @@ class Semgrep < Formula
     sha256 "9751de4cbb57d4bfbf8fc394e125ed4a2f170fbff3dc3d78abf50be85924f8be"
   end
 
-  resource "ruamel.yaml.clib" do
-    url "https://files.pythonhosted.org/packages/8b/25/08e5ad2431a028d0723ca5540b3af6a32f58f25e83c6dda4d0fcef7288a3/ruamel.yaml.clib-0.2.6.tar.gz"
-    sha256 "4ff604ce439abb20794f05613c374759ce10e3595d1867764dd1ae675b85acbd"
-  end
-
   resource "six" do
```
Comment (bayandin) [inline Formula/semgrep.rb:119]: We should install `six` as a dependency instead of a resource.

**Comment by brendongo** [comment]
hmm closing in favor of https://github.com/Homebrew/homebrew-core/pull/90673 but still need to look into this pipenv dependency

**Comment by brendongo** [comment]
Well I can't close this PR but will be fixing in PR linked above

**Pull request closed without merging**